awx_collection/plugins/modules/tower_send.py

```diff
@@ -60,7 +60,7 @@
   - six.moves.StringIO
   - sys
 
-extends_documentation_fragment: awx.awx.auth
+extends_documentation_fragment: awx.awx.auth_legacy
```

Comment:
so the idea is that this won't match it?

https://github.com/ansible/awx/blob/c61e875bf83b60587b1ced02ca3229a36794bda3/awx_collection/tools/roles/template_galaxy/tasks/main.yml#L22

Reply:
No, we should change that too, I dropped the $ so that it will match.